cmd/template/cluster/provider/azure.go

```diff
@@ -62,10 +66,30 @@ func WriteCAPZTemplate(out io.Writer, config ClusterCRsConfig) error {
 		VMSize:            "Standard_D4s_v3",
 	}
 
-	t := template.Must(template.New(config.FileName).Parse(azure.GetTemplate()))
-	err = t.Execute(out, data)
-	if err != nil {
-		return microerror.Mask(err)
+	for _, t := range azure.GetTemplates() {
+		te := template.Must(template.New(config.FileName).Parse(t))
+		buf := new(bytes.Buffer)
+		// Template from our inputs.
+		err = te.Execute(buf, data)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		// DryRun the CR against the management cluster.
+		tYaml, err := runMutation(ctx, client, buf.Bytes(), config.Namespace)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		// Write the yaml to our file.
+		_, err = out.Write(tYaml)
```

Comment:
The output is currently not very pretty - it contains a lot of fields which are not really nice for users to read (e.g. `managedFields`. Example:
```
...
---
apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
kind: AzureCluster
metadata:
  creationTimestamp: "2021-09-09T14:14:40Z"
  generation: 1
  labels:
    cluster.x-k8s.io/cluster-name: yt94h
    cluster.x-k8s.io/watch-filter: capi
    giantswarm.io/cluster: yt94h
    giantswarm.io/organization: giantswarm
    release.giantswarm.io/version: 20.0.0
  managedFields:
  - apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:labels:
          .: {}
          f:cluster.x-k8s.io/cluster-name: {}
          f:cluster.x-k8s.io/watch-filter: {}
          f:giantswarm.io/cluster: {}
          f:giantswarm.io/organization: {}
          f:release.giantswarm.io/version: {}
      f:spec:
        .: {}
        f:location: {}
        f:networkSpec:
          .: {}
          f:vnet:
            .: {}
            f:name: {}
        f:resourceGroup: {}
    manager: kubectl-gs
    operation: Update
    time: "2021-09-09T14:14:40Z"
  name: yt94h
  namespace: org-giantswarm
  uid: 9e2b7d63-30e1-4684-b96d-d693e9626d98
spec:
  location: ""
  networkSpec:
    vnet:
      name: yt94h-vnet
  resourceGroup: yt94h
...
```

Reply:
This is solved now.